zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -1932,6 +1943,11 @@ public void onSuccess(List<JobManagerService.NoteJobInfo> notesJobInfo,
       connectionManager.broadcast(JobManagerServiceType.JOB_MANAGER_PAGE.getKey(),
           new Message(OP.LIST_UPDATE_NOTE_JOBS).put("noteRunningJobs", response));
     }
+
+    @Override
+    public void onFailure(Exception ex, ServiceContext context) {
+      LOGGER.debug(ex.getMessage());
```

Comment:
The parent method should still be called so that other exceptions continue to be logged and handled.
```suggestion
      if (ex instanceof JobManagerForbiddenException) {
          LOGGER.debug(ex.getMessage());
      } else {
          super.onFailure(ex, context);
      }
```